apps/idam/idam-user-dashboard/preview.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: idam-user-dashboard
+  namespace: idam
+spec:
+  values:
+    nodejs:
+      image: hmctspublic.azurecr.io/idam/user-dashboard:prod-ae48887-20211206112137
```

Comment:
is it intentional you aren't configuring any image policies to have this automatically deployed? applies throughout

Reply:
@timja Apology for the delay in replying. I was waiting to discuss with the current IDAM team on whether we need the special preview environment as this new application potentially need the to refer to the  new changes IDAM api on preview. We decided not to do that for now hence removing all the special policies